dom/events/scrolling/wheel-event-groups-target-resize.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+<meta name="viewport" content="width=device-width,initial-scale=1,minimum-scale=1">
+<meta name="timeout" content="long">
+<link rel="stylesheet" type="text/css" href="/fonts/ahem.css"/>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-actions.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script src="scroll_support.js"></script>
+<style>
+body {
+  margin: 0;
+  padding: 0;
+  height: 200vh;
+}
+
+.spacer {
+  width: 100%;
+  height: 25px;
+  padding: 0;
+  margin: 0;
+  font: 25px/1 Ahem;
+}
+
+</style>
+<head>
+<body onload=runTest()>
+<div id="initialTarget" class="spacer" style="background: red"></div>
+<div id="firstRootSpacer" class="spacer" style="background: green"></div>
+<div id="secondRootSpacer" class="spacer" style="background: blue"></div>
+</body>
+
+<script>
+
+function runTest() {
+  promise_test(async (t) => {
+    await waitForCompositorCommit();
+
+    // Wait some extra time to ensure nothing from prior tests impacts this one.
+    // At a minimum this should be greater than 500ms to ensure that the wheel events
+    // in this test are in a new wheel event group
+    await new Promise(resolve => step_timeout(resolve, 1000));
+
+    await waitForCompositorCommit();
+
+    let wheelEventTargets = [];
+    let resizedInitialTarget = false;
+
+    // Resize the initial element the wheel event is targetted at once we fire the
+    // first wheel event, then log the subsequent wheel event targets.
+    function resizeInitialElement(e) {
+      wheelEventTargets.push(e.target);
+      if (!resizedInitialTarget) {
+        resizedInitialTarget = true;
+        e.target.style.height = '10px';
+      }
+    }
+    window.addEventListener("wheel", resizeInitialElement, {passive: true});
```

Comment:
I wonder, `passive` value could change the behavior due to optimization in some browsers. Did you check the `false` case?

Could be nicer to check both `passive: true` and `passive: false` cases in variants.